recordings-downloader/nodejs/index.js

```diff
@@ -4,142 +4,171 @@ const prompt = require('prompt');
 const http = require('https');
 const fs = require('fs');
 
+let batchRequestBody = {
+    batchDownloadRequestList: []
+};
+
 // Set Genesys Cloud objects
 const platformClient = require('purecloud-platform-client-v2');
```

Comment:
I just noticed there's not a package.json file in the nodejs folder. Can you add one so the dependencies can be be installed using `npm i`?